SU-SWS/caw_profile
# CAW21-85 Benefits content type

# READY FOR REVIEW

# Summary
- Benefits content type, fields, and migration
- acceptance tests, displays and More functionality coming later.

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. checkout this branch
2. `blt drupal:update --site=cardinalatwork`
3. go to create a benefit `/node/add/caw_benefits`
4. Fill in some fields
5. verify it saves.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
